src/elarm_server.erl

```diff
@@ -549,6 +558,15 @@ handle_get_alarms(#state{ alarmlist_cb = AlCB,
     {Result, NewAlState} = AlCB:get_alarms(AlState),
```

Comment:
Remove the dynamic function call on line 558. Only modules that define callbacks should make dynamic calls.